### ISSUE TITLE
Enhance image_processing.py

### ISSUE THREAD

**Issue by KBx4** [KBx4/Image-recognition-using-TensorFlow#2]
### It would be a great help if anyone contributes in enhancing the code which could yield better outcomes.

_Workflow:_
- Importing libraries:
  - keras from tensorflow
  - mathplotlib
  - numpy

- Loading fashion_mnist dataset from Keras
- Train-test splitting
- Creating  three-layer neural network using Keras
- Fitting the model with train data
- Evaluating the model using test data
- Printing accuracy
- Making the model predict class of selected image

**Comment by SarathKumaar** [comment]
**Question**
Apart from improving the model output, what other improvements would you like? Also, are there any limitations to keep in mind like model size, memory etc.?

* I'll have a look at it later tonight and tomorrow. Will try to add what I can.

**Comment by KBx4** [comment]
> **Question**
> Apart from improving the model output, what other improvements would you like? Also, are there any limitations to keep in mind like model size, memory etc.?
> 
> * I'll have a look at it later tonight and tomorrow. Will try to add what I can.

You can modify the code as you want, maybe you can add another model if you feel with more complex neural networks or something, anything that you feel like, I am just looking forward to learn more.

Thanks!
